pkg/sink/sink.go

```diff
@@ -153,11 +153,12 @@ func (r Sink) processTrigger(t *triggersv1.EventListenerTrigger, request *http.R
 	return nil
 }
 
-func (r Sink) executeInterceptors(t *triggersv1.EventListenerTrigger, request *http.Request, event []byte, eventID string, log *zap.SugaredLogger) ([]byte, http.Header, error) {
+func (r Sink) executeInterceptors(t *triggersv1.EventListenerTrigger, in *http.Request, event []byte, eventID string, log *zap.SugaredLogger) ([]byte, http.Header, error) {
```

Comment:
I have a few comments about this line (not about this PR specifically)
1. I noticed that we aren't using anything from the `t *triggersv1.EventListenerTrigger` except its Interceptors (`t.Interceptors`), so I think that the function parameter should just be `interceptors []*EventInterceptor` instead of the entire `t *triggersv1.EventListenerTrigger`.
2. I don't think that the `eventID` is used in the `executeInterceptors()` function, so we can remove it as a parameter.
3. The `event []byte` is already in the `in *http.Request` parameter, so it's duplicate information in the `executeInterceptors()` function. I think that we could clean up the function declaration a little bit by removing the `event []byte` parameter.

Cleaning up these parameters will make is cleaner and easier to test the function, because we won't have to create as many fake parameters to feed into our tests.

Reply:
Agreed! I tackled most of this in #326, but looks like I forgot to bubble this up to the event listener. I can tackle this in another change.

Reply:
Refactoring in a different PR sounds good to me. Any other concerns @ncskier ?

Reply:
Nope, as long as it's handled in another PR, should be good; thanks! 👍